### ISSUE TITLE
ego-rnn  をpytorch 1.4.0で動かすメモ

### ISSUE THREAD

**Issue by idekazuki** [idekazuki/diary#100]
pyenv/detec2/lib/python3.7/site-packages/torch/optim/lr_scheduler.py:122: UserWarning: Detected call of `lr_scheduler.step()` before `optimizer.step()`. In PyTorch 1.1.0 and later, you should call them in the opposite order: `optimizer.step()` before `lr_scheduler.step()`.  Failure to do this will result in PyTorch skipping the first value of the learning rate schedule. See more details at https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate
  "https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate", UserWarning)
/home/yanai-lab/ide-k/ide-k/pyenv/detec2/lib/python3.7/site-packages/torch/nn/functional.py:1351: UserWarning: nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.
  warnings.warn("nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.")

直さなくても動いたが、心配なのでoptimiser.step()をlr_schedule()の前に移動させることにした。

他には細かいところでcuda() Tensorに変更
loss[0] のを[0]削除

**Comment by idekazuki** [comment]
実行コマンド
`python main-run-rgb.py --dataset gtea61 --stage 1 --trainDatasetDir /home/yanai-lab/ide-k/ide-k/gtea_61 --outDir experiments --seqLen 25 --trainBatchSize 32 --numEpochs 300 --lr 1e-3 --stepSize 25 75 150 --decayRate 0.1 --memSize 512`
